src/main/java/com/glencoesoftware/bioformats2raw/MiraxReader.java

```diff
@@ -1069,10 +1069,17 @@ public void setFillValue(Byte fill) {
    * for brightfield.
    *
    * @return fill value for missing tiles
+   * @deprecated use getFillColor()
    */
   public byte getFillValue() {
-    if (fillValue != null) {
-      return fillValue;
+    return getFillColor();
+  }
+
+  @Override
+  public Byte getFillColor() {
+    Byte color = getFillColor();
```

Comment:
In my testing this caused a `java.lang.StackOverflowError: null` due to the infinite recursion

Reply:
Should be fixed with e3dd7ca.